components/discord_bot/actions/list-guild-members/list-guild-members.mjs

```diff
@@ -0,0 +1,39 @@
+import common from "../common.mjs";
```

Comment:
Since this component doesn't use `channelId`, you can include common.mjs from the discord_bot directory.
`import common from "../../common.mjs"`;

Reply:
In this case I had to do it that way since the `description` for `guildId` is a bit different

Reply:
Yep, you're right. You should leave in `guildId` so you can overwrite the description. This action doesn't use `channelId` though, right? So you could import `"../../common.mjs"` instead of `"../common.mjs"`.

```
props: {
   ....common.props,
   guildId: {
```

Reply:
Was the props section updated to include `...common.props` and remove `discord`? I'm not seeing that change.

Reply:
yeah but remember the description here is a bit different

Reply:
Yes, you want to overwrite guildId to add the new description. However, this component is still inheriting from common, so when you're overwriting the props section, you should add back in any base props from common with the line `...common.props,`. Then, since `discord` is a prop in common, you don't need to list it again in the component props.

Reply:
Yeah I totally missed that, thanks!